### PR TITLE
Move EIP-2159 to last call

### PR DESCRIPTION
**Pull request by ajsutton**
The implementation of EIP-2159 has shipped in Pantheon and the technical details are all stable.

As such, moving it to last call status.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 2159 state from Draft to Last Call

File: EIPS/eip-2159.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ eip: 2159
 title: Common Prometheus Metrics Names for Clients
 author: Adrian Sutton (@ajsutton)
 discussions-to: https://ethereum-magicians.org/t/common-chain-metrics/3415
-status: Draft
+status: Last Call
```
Comment (axic) [inline EIPS/eip-2159.md]: This status needs a `review-period-end` field.

Comment (ajsutton) [reply EIPS/eip-2159.md]: Thank you, I knew I was missing something...

**Comment by ajsutton** [comment]
The build appears to have failed while installing yum dependencies - maybe a network blip or something?  Could it be kicked to run again?

**Pull request merged**